editor/queries/ems.ts

```diff
@@ -20,10 +24,113 @@ export const GET_EMS_RECORD = gql`
       pcr_patient_age
       pcr_patient_gender
       pcr_patient_race
+      person_id
       travel_mode
+      unparsed_apd_incident_numbers
       crash {
         cris_crash_id
       }
     }
   }
 `;
+
+export const GET_MATCHING_PEOPLE = gql`
+  query EMSMatchingCrashes($crash_pks: [Int!]) {
+    people_list_view(
+      where: { crash_pk: { _in: $crash_pks } }
+      order_by: { cris_crash_id: asc, unit_nbr: asc, prsn_nbr: asc }
+    ) {
+      crash_pk
+      crash_timestamp
+      id
+      unit_nbr
+      is_primary_person
+      prsn_age
+      drvr_city_name
+      drvr_zip
+      prsn_exp_homelessness
+      prsn_first_name
+      prsn_mid_name
+      prsn_last_name
+      prsn_injry_sev_id
+      injry_sev {
+        id
+        label
+      }
+      prsn_type_id
+      prsn_type {
+        id
+        label
+      }
+      prsn_gndr_id
+      gndr {
+        id
+        label
+      }
+      prsn_ethnicity_id
+      drvr_ethncty {
+        id
+        label
+      }
+      crash {
+        id
+        cris_crash_id
+        record_locator
+        case_id
+        crash_timestamp
+        fhe_collsn_id
+        collsn {
+          id
+          label
+        }
+        address_primary
+        address_secondary
+        latitude
+        longitude
+        location_id
+      }
+      unit {
+        id
+        unit_nbr
+        veh_mod_year
+        unit_desc_id
+        unit_desc {
+          id
+          label
+        }
+        veh_body_styl_id
+        veh_body_styl {
+          id
+          label
+        }
+      }
+    }
+  }
+`;
+
+export const UPDATE_EMS_INCIDENT_CRASH_AND_PERSON = gql`
```

Comment:
You mentioned a separate unhandled graphql error so I wanted to mention one related to this mutation too. When matching an unmatched EMS patient, the rows in the associated people records table still shows an enabled select button next to people records that have already been matched in the UI. 

There might be plans to address this beyond this PoC but didn't want to let it slip by.

https://github.com/user-attachments/assets/c90d21ca-f735-4d7b-a0f9-f3f928094e0e